Automattic/wp-calypso
# [WordPress → Jetpack app branding] Update mobile card on home page

_Context:_ Following the plans to [refocus the WordPress apps on core features](https://make.wordpress.org/mobile/2022/07/27/refocusing-the-wordpress-app-on-core-features/), we plan to move features that are specific to WordPress.com to the Jetpack app. As such, we will eventually replace all mentions of the WordPress app in Calypso with Jetpack.

#### Proposed Changes

* This PR builds on the work done in https://github.com/Automattic/wp-calypso/pull/68062, which changed the home page's app CTA in mobile browsers so that it links to the Jetpack app. Following this PR, the updated CTA will also display in desktop browsers.
* The conditionals that limited the new CTA to mobile browsers have been removed, and some styles have been slightly tweaked to ensure the CTA matches the design for desktop browsers. 

#### Screenshots

<details>
 <summary>Design / Actual ⤵️</summary>

| Design  | Actual |
| ------------- | ------------- |
| <img src="https://user-images.githubusercontent.com/2998162/195420746-ca6e9167-dc10-4f7c-a8cf-edb45fd01bbf.png" width="100%"> | <img src="https://user-images.githubusercontent.com/2998162/195420646-984564f9-5b4f-43a5-b29e-634eded3be02.png" width="100%"> |

</details>

<details>
 <summary>Before / After ⤵️</summary>

| Before  | After |
| ------------- | ------------- |
| <img src="https://user-images.githubusercontent.com/2998162/195420503-29d892e0-bb66-4209-874b-e9fe0a47834f.png" width="100%"> | <img src="https://user-images.githubusercontent.com/2998162/195420470-18dfed2d-c4b5-4d8f-ba99-b5d5a5b4dbb9.png" width="100%"> |

</details>

#### Testing Instructions

* Navigate to the customer home page and verify that the new CTA displays in the right sidebar.
* In addition, verify that there have been no changes to the way the CTA displays in mobile browsers.

#### Pre-merge Checklist






Related to #
